instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py

```diff
@@ -73,21 +73,23 @@ def started(self, event: monitoring.CommandStartedEvent):
             span = self._tracer.start_span(name, kind=SpanKind.CLIENT)
```

Comment:
In terms of the name, I think we can remove the "DATABASE_TYPE" portion and just use the `command_name`.

Reply:
EDIT: We can leave these in until the specs are finalized.